doc/user/content/get-started.md

```diff
@@ -73,237 +62,465 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
+     id       | f        | bigint
+     seller   | f        | bigint
+     item     | f        | text
+     end_time | f        | timestamp with time zone
+    (4 rows)
+    ```
 
     ```sql
-    SHOW SOURCES;
+    SHOW COLUMNS FROM bids;
     ```
+    <p></p>
 
     ```nofmt
-       name    | type  |  size
-    -----------+-------+---------
-     items     | kafka | 3xsmall
-     purchases | kafka | 3xsmall
+        name    | nullable |           type
+    ------------+----------+--------------------------
+     id         | f        | bigint
+     buyer      | f        | bigint
+     auction_id | f        | bigint
+     amount     | f        | integer
+     bid_time   | f        | timestamp with time zone
+    (5 rows)
     ```
 
-    Creating a source will prompt Materialize to **start ingesting data** from the specified topics into durable storage.
+## Step 4. Compute real-time results
 
-### Transform data
+With auction data streaming in, you can now explore the unique value of Materialize: computing real-time results over fast-changing data.
 
-With data being ingested into Materialize, we can start building SQL statements to transform it into something actionable.
+1. Switch to your `compute_qck` cluster:
 
-1. Create a [view](/overview/key-concepts/#non-materialized-views) `item_purchases` that calculates rolling aggregations for each item in the inventory:
+    ```sql
+    SET CLUSTER = compute_qck;
+    ```
+
+1. Enable `psql`'s timing feature so you can see how quickly results are returned:
 
     ```sql
-    CREATE VIEW item_purchases AS
-      SELECT
-        item_id,
-        SUM(quantity) AS items_sold,
-        SUM(purchase_price) AS revenue,
-        COUNT(id) AS orders,
-        MAX(created_at::timestamp) AS latest_order
-      FROM purchases
-      GROUP BY item_id;
+    \timing
     ```
 
-    As in other databases, a view doesn't store the results of the query, but simply provides an alias for the embedded `SELECT` statement.
+1. Let's start by creating a [**view**](/overview/key-concepts/#non-materialized-views):
 
-    To keep the results of this view incrementally maintained **in memory** within our cluster, we'll create an [index](/sql/create-index).
+    ```sql
+    CREATE VIEW avg_bids AS
+      SELECT auctions.item, avg(bids.amount) AS average_bid
+      FROM bids
+      JOIN auctions ON bids.auction_id = auctions.id
+      WHERE bids.bid_time < auctions.end_time
+      GROUP BY auctions.item;
+    ```
 
-1. Create an index `item_purchases_idx`:
+    This view joins data from `auctions` and `bids` to get the average price of bids eligible to be winners, that is, bids that arrived before their auctions closed.
 
-   ```sql
-   CREATE INDEX item_purchases_idx ON qck.item_purchases (item_id);
-   ```
+    Note that, as in other SQL databases, a view in Materialize is simply an alias for the embedded `SELECT` statement. The results of the query are computed only when the view is queried.
```

Comment:
```suggestion
    Note that, as in other SQL databases, a view in Materialize is an alias for the embedded `SELECT` statement. Materialize computes the results of the query only when you query the view.
```

Reply:
Done. Thanks for helping to fix some passive voice in here. I think the second case is misleading, though, since it suggests that user needs to be involved, when it can be a client, etc.